scripts/zones/Bibiki_Bay/mobs/Shen.lua

```diff
@@ -3,24 +3,111 @@
 --  Mob: Shen
 -----------------------------------
 local ID = require("scripts/zones/Bibiki_Bay/IDs")
-require("scripts/mixins/families/uragnite")
 -----------------------------------
 local entity = {}
 
+local function enterShell(mob)
+    mob:setAnimationSub(1)
+    mob:SetAutoAttackEnabled(false)
+    mob:SetMagicCastingEnabled(false)
+    mob:setMod(xi.mod.UDMGPHYS, -8500)
```

Comment:
mmmkkkaaaay. This is the [2nd time](https://github.com/LandSandBoat/server/pull/2158#issuecomment-1173310670) recently I've seen this done with slightly diff values from the mixin. I really feel we should adjust the mixin to allow for some parameters to control it, rather than having the same code with diff input in multiple places.

Reply:
I guess I personally found it cleaner to just put the logic in Shen's code so I don't have to clog up the mixin with several checks to see if its a certain mob or not. The only thing Shen shares with the mixin is the enter/exit shell portion.

Reply:
we wouldn't want check if its a particular mob, we'd want input for how potent the dmg reductions will be if diff from defaults. other stuff would still be separate if needed, but reduces down to just checking animation sub

Reply:
So I took another look at this, and I still feel like it would be best to not utilize the mixin for Shen. Shen does not follow the same behavior the mixin uses at all other than the enter/exit shell function, which I need to call directly in Shen's lua to behave properly.